tests/dummy/app/templates/docs/route-based-targeting.md

```diff
@@ -0,0 +1,133 @@
+# Route-based Targeting
+
+You can use the functions provided by this addon to set up [page-level GPT key-value targeting](https://support.google.com/admanager/answer/1697712#use_key-value_targeting_with_GPT) in your application.
+
+## Setting targeting for the path
+
+We usually want to add key-values based on the url to allow for targeting ads by path.
+
+To set up targeting for your path that will be applied on all routes in your application, add `doPathTargeting` and `clearPathTargeting` to the `didTransition` and `willTransition` actions on your `routes/application.js`.
```

Comment:
include a note that this requires that the did/willTransition events need to bubble, i.e. sub-routes should not return false from their own did/willTransition events, if they implement any